src/Input/Input.js

```diff
@@ -67,6 +67,12 @@ export const styleSheet = createStyleSheet('MuiInput', (theme) => {
         appearance: 'none',
       },
     },
+    multiLine: {
+      resize: 'none',
+      'line-height': 'inherit',
+      padding: '0px',
+      'margin-top': '12px',
```

Comment:
Yes indeed, isn't that better? :) 
The Safari style can be fixed by adding a margin 0 on the input.